src/main/java/com/mastery/java/task/dao/EmployeeDao.java

```diff
@@ -0,0 +1,24 @@
+package com.mastery.java.task.dao;
+
+import com.mastery.java.task.dto.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+
+@Repository
+public interface EmployeeDao extends JpaRepository<Employee, Integer> {
+
+    public Employee getEmployeeByEmployeeId(int employeeId);
+
+    @Query(value = "SELECT employee_id FROM employee WHERE first_name = :firstName AND last_name = :lastName AND departament_id = :departamentId " +
+            "AND job_title = :jobTitle AND gender = :gender AND date_of_birth = :dateOfBirth", nativeQuery = true)
+    public Integer getExistingEmployee(@Param("firstName") String firstName,
```

Comment:
why don't you use the Spring Data  Query Creation? https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.query-creation

Reply:
i don't want to get the whole entity from the database, i only need the number if one exists

Reply:
ok then, no worries mate